nilearn/maskers/nifti_labels_masker.py

```diff
@@ -89,6 +90,22 @@ class NiftiLabelsMasker(BaseMasker, _utils.CacheMixin):
         Must be one of: sum, mean, median, minimum, maximum, variance,
         standard_deviation. Default='mean'.
 
+    keep_masked_labels : :obj:`bool`, optional
+        When a mask is supplied through the "mask_img" parameter, some
+        labels in the atlas may not have any brain coverage within the
```

Comment:
I think we can still improve the docstring a little bit. maybe instead of "may not have any brain coverage within the masked region" we could say "some atlas regions may lie entirely outside of the brain mask"?